dj_gui_api_server/DJConnector.py

```diff
@@ -96,12 +95,12 @@ def list_tables(jwt_payload: dict, schema_name: str):
                     table_name_parts[-2]) + '.' + DJConnector.snake_to_camel_case(
                         table_name_parts[-1]))
             else:
-                print(table_name + ' is of unknown table type')
+                print(table_name + ' is of unknown table type', flush=True)
```

Comment:
Why not simply raise an exception here with this detail? API side already is catching it and returning successfully with detail to frontend. Would prefer that model so we are more likely catch issues related to unexpected table tiers (though they all seem to be accounted) and users can provide feedback that will lead to a better resolution.

Reply:
This was probably a relic from when I first started on the api and didn't have error handling yet.

Reply:
Solved